Valet/VALValet.m

```diff
@@ -428,11 +428,17 @@ - (BOOL)setObject:(NSData *)value forKey:(NSString *)key options:(NSDictionary *
     
     __block OSStatus status = errSecNotAvailable;
     VALExecuteBlockInLock(^{
+#if TARGET_OS_IPHONE
```

Comment:
Kinda difficult to read the control flow with this preprocessor stuff. Either some commments or something else may help.